### PR TITLE
Append appIdentifier to get membership method

### PR DESCRIPTION
**Pull request by aligator3**

### CODE REVIEW

File: Navis.SDK.CompanyCloud/Clients/AccountMembershipClient.cs [hunk 1]
```diff
@@ -21,13 +21,14 @@ public AccountMembershipClient(ITokenProvider tokenProvider, INcsSettings ncsSet
         /// <param name="accountIdentifier">The account identifier which can be domain or Uid.</param>
         /// <param name="cancellationToken">A cancellation token that can be used by other
         /// objects or threads to receive notice of cancellation.</param>
+        /// <param name="appIdentifier">App identifier to fetch only app-related features</param>
         /// <exception cref="HttpException">A server side error occurred.</exception>
         public async Task<DTO.Query.AccountMembership> GetByIdAsync(string accountIdentifier,
-            CancellationToken cancellationToken)
+            CancellationToken cancellationToken, string appIdentifier = null)
```
Comment (carsten-krueger) [inline Navis.SDK.CompanyCloud/Clients/AccountMembershipClient.cs]: Cancellation token should be the last parameter.

File: Navis.SDK.CompanyCloud/Clients/AccountMembershipClient.cs [hunk 2]
```diff
@@ -21,13 +21,14 @@ public AccountMembershipClient(ITokenProvider tokenProvider, INcsSettings ncsSet
...
         {
             const string route = "/v1/membership";
             var result = await GetObjectAsync<DTO.Query.AccountMembership>(accountIdentifier, null,
-                route, cancellationToken);
+                route, cancellationToken, appIdentifier);
```
Comment (carsten-krueger) [inline Navis.SDK.CompanyCloud/Clients/AccountMembershipClient.cs]: Same here.

File: Navis.SDK.CompanyCloud/Core/ApiWrapper.cs [hunk 3]
```diff
@@ -49,9 +49,10 @@ protected virtual void UpdateJsonSerializerSettings(JsonSerializerSettings setti
         /// <param name="route">Route to use to post object.</param>
         /// <param name="cancellationToken">A cancellation token that can be used by other
         /// objects or threads to receive notice of cancellation.</param>
+        /// <param name="appIdentifier">App identifier to fetch only app-related features</param>
         /// <returns>Success</returns>
         /// <exception cref="HttpException">A server side error occurred.</exception>
-        public async Task<T> GetObjectAsync<T>(string accountIdentifier, string apiKey, string route, CancellationToken cancellationToken)
+        public async Task<T> GetObjectAsync<T>(string accountIdentifier, string apiKey, string route, CancellationToken cancellationToken, string appIdentifier = null)
```
Comment (carsten-krueger) [inline Navis.SDK.CompanyCloud/Core/ApiWrapper.cs]: Same here.

**Review by carsten-krueger — CHANGES_REQUESTED** [review]

File: Navis.SDK.CompanyCloud/Clients/AccountMembershipClient.cs (hunk 1 above)
Comment (aligator3) [reply Navis.SDK.CompanyCloud/Clients/AccountMembershipClient.cs]: done

**Review by carsten-krueger — APPROVED** [review]

**Pull request merged**